Polyhedron/demo/Polyhedron/Plugins/Display/Display_property_plugin.cpp

```diff
@@ -809,6 +863,109 @@ private Q_SLOTS:
   }
 
+private Q_SLOTS:
+  void setExpandingRadius()
+  {
+    double sliderMin = dock_widget->expandingRadiusSlider->minimum();
+    double sliderMax = dock_widget->expandingRadiusSlider->maximum() - sliderMin;
+    double val = dock_widget->expandingRadiusSlider->value() - sliderMin;
+    sliderMin = 0;
+
+    Scene_item* item = scene->item(scene->mainSelectionIndex());
+    Scene_surface_mesh_item* sm_item = qobject_cast<Scene_surface_mesh_item*>(item);
+    if(sm_item == nullptr)
+      return;
+
+    SMesh& smesh = *(sm_item->face_graph());
+
+    auto vpm = get(CGAL::vertex_point, smesh);
+
+    // @todo use the upcoming PMP::longest_edge
+    if(maxEdgeLength < 0)
+    {
+      auto edge_range = CGAL::edges(smesh);
+
+      if(num_edges(smesh) == 0)
+      {
+        expand_radius = 0;
+        dock_widget->expandingRadiusLabel->setText(tr("Expanding Radius: %1").arg(expand_radius));
+        return;
+      }
+
+      auto eit = std::max_element(edge_range.begin(), edge_range.end(),
+                                  [&, vpm, smesh](auto l, auto r)
+                                  {
+                                    auto res = EPICK().compare_squared_distance_3_object()(
+                                        get(vpm, source((l), smesh)),
+                                        get(vpm, target((l), smesh)),
+                                        get(vpm, source((r), smesh)),
+                                        get(vpm, target((r), smesh)));
+                                    return (res == CGAL::SMALLER);
+                                });
+
+      CGAL_assertion(eit != edge_range.end());
+
+      maxEdgeLength = PMP::edge_length(*eit, smesh);
+    }
+
+    double outMax = 5 * maxEdgeLength, base = 1.2;
+
+    expand_radius = (pow(base, val) - 1) * outMax / (pow(base, sliderMax) - 1);
+    dock_widget->expandingRadiusLabel->setText(tr("Expanding Radius: %1").arg(expand_radius));
+  }
+
+private:
+  void displayInterpolatedCurvatureMeasure(Scene_surface_mesh_item* item,
+                                           CurvatureType mu_index)
+  {
+    if(mu_index != MEAN_CURVATURE && mu_index != GAUSSIAN_CURVATURE)
+      return;
+
+    std::string tied_string = (mu_index == MEAN_CURVATURE) ? "v:display_plugin_interpolated_corrected_mean_curvature"
+                                                           : "v:display_plugin_interpolated_corrected_Gaussian_curvature";
+
+    SMesh& smesh = *item->face_graph();
+
+    const auto vnm = smesh.property_map<vertex_descriptor, EPICK::Vector_3>("v:normal_before_perturbation").first;
+    const bool vnm_exists = smesh.property_map<vertex_descriptor, EPICK::Vector_3>("v:normal_before_perturbation").second;
+
+    // compute once and store the value per vertex
+    bool non_init;
+    SMesh::Property_map<vertex_descriptor, double> mu_i_map;
+    std::tie(mu_i_map, non_init) = smesh.add_property_map<vertex_descriptor, double>(tied_string, 0);
```

Comment:
More a general remark (@sloriot ): it seems in the demo we add properties,but do not delete them.